docs/atlascli/command/atlas-dbusers-create.txt

```diff
@@ -53,7 +55,7 @@ Options
    * - --awsIAMType
      - string
      - false
-     - AWS IAM method by which the provided username is authenticated. Valid values are NONE, USER, or ROLE. This value defaults to "NONE".
+     - AWS IAM method by which the provided username is authenticated. Valid values are NONE, USER, or ROLE. If you set this to USER or ROLE, the user authenticates with IAM credentials and doesn't need a password. If you set this to USER or ROLE, you can't set --x509Type or --ldapType to any value other than NONE. This value defaults to "NONE".
```

Comment:
Question: does mcli allow formatting of this as a bulleted list, like this:

```

This value defaults to "NONE". If you set this value to USER or ROLE:

- The user authenticates with AWS IAM credentials and doesn't need a password.
- You can't set --x509Type or --ldapType to any value other than NONE. 

```

Reply:
No unfortunately